core/services/bulletprooftxmanager/eth_broadcaster.go

```diff
@@ -228,6 +230,13 @@ func (eb *ethBroadcaster) processUnstartedEthTxs(fromAddress gethCommon.Address)
 	}
 }
 
+func (eb *ethBroadcaster) initialTxGasPrice() *big.Int {
+	if eb.config.OptimismGasFees() {
+		return big.NewInt(1)
```

Comment:
@j16r this is 1 gwei or 1 wei? 

It should be 1 gwei, or 1000000000 wei.

Reply:
Resolved: [Optimism gas price should be 1 GWei #4285](https://github.com/smartcontractkit/chainlink/pull/4285)